dockerfiles/init/modules/che/templates/che.env.erb

```diff
@@ -23,7 +22,8 @@ JPDA_ADDRESS=<%= scope.lookupvar('che::che_debug_port') %>
 
 CHE_DATABASE=/data
 CHE_WORKSPACE_STORAGE=/data/workspaces
-CHE_WORKSPACE_LOGS=/data/logs/machine
+CHE_LOGS_DIR=/logs
```

Comment:
@garagatyi I've moved the definition few lines after its initial location so the LOGS_DIR are at the same place, I've not introduced a new hardcoded property.

AFAIK the definition of these values are "hardcoded" by CLI as it's inside the docker container. (for Che and Codenvy)